server/_dbConnect.ts

```diff
@@ -17,7 +17,6 @@ export const dbConnect = async (): Promise<MongoClient | undefined> => {
     await client.connect()
     // Establish and verify connection
     await client.db('admin').command({ ping: 1 })
```

Comment:
I don't think we need to use the admin db for this verification, i did this in early testing. 
We should be able to ping the db we are using